test/TokenBalanceOracle.js

```diff
@@ -2,36 +2,48 @@ const { assertRevert } = require('./helpers/helpers')
 const Oracle = artifacts.require('TokenBalanceOracle')
 const MockErc20 = artifacts.require('TokenMock')
 
-import DaoDeployment from './helpers/DaoDeployment'
-import { deployedContract } from './helpers/helpers'
+const deployDAO = require('./helpers/deployDao')
+const { deployedContract } = require('./helpers/helpers')
+const { hash: nameHash } = require('eth-ens-namehash')
 
 const ANY_ADDR = '0xffffffffffffffffffffffffffffffffffffffff'
 
-contract('TokenBalanceOracle', ([rootAccount, ...accounts]) => {
-  let daoDeployment = new DaoDeployment()
+contract('TokenBalanceOracle', ([appManager, account1, account2, nonContractAddress]) => {
```

Comment:
You could define the accounts in terms of their balances to make it easier to interpret tests that refer to the balance within these accounts. eg `accountBal100, accountBal0`. Not necessary but just an idea.